server/server/settings.py

```diff
@@ -96,6 +96,7 @@ def resolver_context_processor(request):
             'context_processors': [
                 'django.contrib.auth.context_processors.auth',
                 'django.template.context_processors.debug',
+                'django.template.context_processors.request',
```

Comment:
Please add a comment explaining why this change is needed (required for specific view from django-notifications IIRC)